content/docs/dvclive/api-reference/next_step.md

```diff
@@ -0,0 +1,52 @@
+# dvclive.next_step()
+
+Signals that the current step has ended. Check the
+[init parameters](/doc/dvclive/api-reference/init#parameters) for configuring
+the behavior.
+
+```py
+def next_step()
+```
+
+#### Usage:
+
+```py
+import dvclive
+
+for step in range(3):
+    dvclive.log("metric", 0.9)
+    dvclive.next_step()
+```
+
+## Description
+
+Each call to `dvclive.next_step` will behave depending on the selected
+[init parameters](/doc/dvclive/api-reference/init#parameters) and whether `DVC`
+is available or not.
```

Comment:
Good Description start here already. But I would just mention briefly that this function uses DVCLive config parameters `summary` and `path` which default to `True` and `dvclive` but can be defined via `init()` (or DVC env but that's also better explained in the `init()` ref.).

Probably no need to complicate this ref with the DVC integration info, as it can be covered in it's own section of the description ⏳